.vsts-ci.yml

```diff
@@ -62,7 +62,7 @@ stages:
         pool:
           name: NetCorePublic-Pool
           queue: BuildPool.Windows.10.Amd64.VS2019.Pre.Open
-        helixTargetQueue: Windows.10.Amd64.ClientRS5.VS2019.BT.Open
+        helixTargetQueue: buildpool.server.amd64.vs2019.bt.open
```

Comment:
Oops I didn't realize what you were doing here.  

Couple points:
- Buildpool queues only are for, well, builds..  If you're concerned about perf (especially with internal), you can and should move to these.
- While there indeed are server-based non-build Helix SKUs,  I don't think we've stood up any with VS on them because that's not something we've intended to do.